app/models/application_form.rb

```diff
@@ -6,10 +6,15 @@ class ApplicationForm < ApplicationRecord
   has_many :application_work_experiences
   has_many :application_volunteering_experiences
   has_many :application_qualifications
+  has_many :references
```

Comment:
Do we also need some validation to force candidates to enter two referees before submitting their application? Or is this something that they can return to after submitting? Thinking about it the later makes more sense to I guess the validation is unhelpful here.

Reply:
There are a couple of non-trivial things with that requirement – could we pick this off on a subsequent PR?